### ISSUE TITLE
Confidence values for predictions?

### ISSUE THREAD

**Issue by olix20** [cmusphinx/g2p-seq2seq#131]
At test time, some of the words could be quite new and radically different from the set of words used for training the model. Also there could be samples in the training set which are mislabelled. I'm looking for a metric that could help me identify such cases; something like raw softmax values per step to identify parts of output which could be outlier-ish or min/average of them as a proxy for confidence. 

Can i check if you've considered this before, if it makes sense, and what are some ways to extract those?

**Comment by Danial-Alh** [comment]
It's maybe a bit irrelevant to this topic but I was wondering if you could report confidences in the beam search result; that's useful there too.
